src/network/networkd-setlink.c

```diff
@@ -483,6 +515,15 @@ static bool link_is_ready_to_call_set_link(Request *req) {
                         }
                 }
                 break;
+        case SET_LINK_MAC:
+                if (PTR_TO_INT(req->userdata)) {
```

Comment:
maybe add a comment here, what this is about. The variable name doesn't tell us anything after all...